### ISSUE TITLE
doomemacs/straigh issue when instlling/running 

### ISSUE THREAD

**Issue by oatmealm** [brunoarine/org-similarity#24]
Doomemacs works with straight and compiles into a sepereate directory per Emacs version in use. To finish the creation of the venv I had to manually perform the steps, and also copy over the setup, pyproject and README files... which resulted in:

```
Successfully built org-similarity
Installing collected packages: org-similarity
Successfully installed org-similarity-0.2
```

When trying `org-similarity-insert-list` it insists missing modules need to be installed, though when running `org-similarity-sidebuffer` it seems to work...

**Comment by brunoarine** [comment]
Thank you @oatmealm for reporting the issue! Could you tell me more details about the problem? Including:

- How is the entry for org-similarity written in your `packages.el` file, and
- Which Emacs version are you using?

**Comment by brunoarine** [comment]
@oatmealm Hello again! There have been a lot of upgrades and bug fixes recently. If you're still interested in using org-similarity, could you kindly check if you're still having this issue with the latest release tag?

**Comment by oatmealm** [comment]
Yes. Actually saw your post on mastodon I think. Made me look and also start playing with whisper independently, which looks promising indeed in terms of accuracy, so curious indeed.
